### PR TITLE
feat: add http cache

### PR DESCRIPTION
**Pull request by loadez**
Added http cache to all backend paths

### CODE REVIEW

File: backend/kernelCI_app/urls.py [hunk 1]
```diff
@@ -1,18 +1,30 @@
+import os
 from django.urls import path
+from django.views.decorators.cache import cache_page
 from kernelCI_app import views
 
+cacheTimeoutEnv = os.environ.get("CACHE_TIMEOUT")
+cacheTimeout = 600 if cacheTimeoutEnv is None else int(cacheTimeoutEnv)
```
Comment (lfjnascimento) [inline backend/kernelCI_app/urls.py:7]: move this env to `settings.py`

Comment (lfjnascimento) [reply backend/kernelCI_app/urls.py:7]: also, try to use `get_json_env_var`

**Review by WilsonNet — COMMENTED** [review]
I approve this but let's wait for after the demo

File: backend/kernelCI_app/urls.py [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
+    path("tree/tests/", cache_page(timeout)
+         (views.groupedTests.as_view()), name="treeGroupedTests"),
+    path("tree/<str:commit_hash>", cache_page(timeout)
+         (views.TreeDetails.as_view()), name="treeDetails"),
     path(
-        "tree/<str:commit_hash>/tests/", views.TreeTestsView.as_view(), name="treeTests"
+        "tree/<str:commit_hash>/tests/",
+        cache_page(timeout)(views.TreeTestsView.as_view()), name="treeTests"
     ),
-    path("build/<str:build_id>", views.BuildDetails.as_view(), name="buildDetails"),
```
Comment (lucasbracher) [inline backend/kernelCI_app/urls.py:13]: I'd rather write this way:
```
path("tree/", cache_page(timeout)(views.TreeView.as_view()), 
    name="tree"),
```
In order to improve readability.
The rest is perfect!

**Review by lfjnascimento — APPROVED** [review]

**Pull request merged**